LinqToDBPostGisNetTopologySuite/SpatialRelationships.cs

```diff
@@ -213,6 +228,52 @@ public static class SpatialRelationships
             throw new InvalidOperationException();
         }
 
+        /// <summary>
+        /// Takes geomA and geomB and returns the DE-9IM
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_Relate.html
+        /// </remarks>
+        /// <param name="geometry">Input geometry</param>
+        /// <param name="other">Input other geometry</param>
+        /// <returns>DE-9IM between input geometries</returns>
+        [Sql.Function("ST_Relate", ServerSideOnly = true)]
+        public static string STRelate(this NTSG geometry, NTSG other)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Takes geomA and geomB and returns the DE-9IM
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_Relate.html
+        /// </remarks>
+        /// <param name="geometry">Input geometry</param>
+        /// <param name="other">Input other geometry</param>
+        /// <param name="BoundaryNodeRule">Boundary node rule (1:OGC/MOD2, 2:Endpoint, 3:MultivalentEndpoint, 4:MonovalentEndpoint)</param>
+        /// <returns>DE-9IM between input geometries</returns>
+        [Sql.Function("ST_Relate", ServerSideOnly = true)]
+        public static string STRelate(this NTSG geometry, NTSG other, int BoundaryNodeRule)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Takes intersectionMatrix and intersectionMatrixPattern and Returns true if the intersectionMatrix satisfies the intersectionMatrixPattern
+        /// </summary>
+        /// <remarks>
+        /// See https://postgis.net/docs/manual-3.0/ST_RelateMatch.html
+        /// </remarks>
+        /// <param name="intersectionMatrix">Input intersectionMatrix</param>
+        /// <param name="intersectionMatrixPattern">Input intersectionMatrixPattern</param>
+        /// <returns>If the intersectionMatrix satisfies the intersectionMatrixPattern</returns>
+        [Sql.Function("ST_RelateMatch", ServerSideOnly = true)]
+        public static string STRelateMatch(this string intersectionMatrix, string intersectionMatrixPattern)
```

Comment:
`this string`  should be just `string`
(not an extension method for `string`)

Reply:
OK.
Also thought about whether to add “this”，but for the Invoking flexibility i kept it here.